frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.mdx

```diff
@@ -0,0 +1,104 @@
+import { Canvas, Meta } from '@storybook/blocks';
+import { Heading, Paragraph } from '@digdir/design-system-react';
+import * as StudioTableLocalPaginationStories from './StudioTableLocalPagination.stories';
+
+<Meta of={StudioTableLocalPaginationStories} />
+
+<Heading level={1} size='small'>
+  StudioTableLocalPagination
+</Heading>
+<Paragraph>StudioTableLocalPagination is a wrapper for StudioTableRemotePagination.</Paragraph>
+<Paragraph>
+  <ul>
+    <li>
+      Use this component when you are fetching a complete set of data, and want the component to
+      handle pagination and sorting internally. Otherwise, use `StudioTableRemotePagination`.
+    </li>
+    <li>
+      Pagination and sorting can be removed by not passing props to `pagination` and setting
+      `isSortable` to `false`.
+    </li>
+  </ul>
+</Paragraph>
+
+<Canvas of={StudioTableLocalPaginationStories.Preview} />
+
+<Heading level={2} size='xsmall'>
+  Type definitions
+</Heading>
+
+```tsx
```

Comment:
I think we don't need to define the type here? It means we have to remember to update and maintain it two places. I suggest removing it, what do you think? 😊

Reply:
Agreed - fixed 😊